style.css

```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+  height: 150vh;
+}
+
+#bar-container {
+  background-color: #3c3a39;
+  padding: 3% 10%;
+}
+
+#bar-container,
+.logo {
+  color: #fff;
+}
+
+#status-bar {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 3%;
+}
+
+.time {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+  width: 54px;
+  height: 16px;
+}
+
+.fas-container .fas {
+  height: 10px;
+}
+
+/* Bar2 Logo and Menu */
+#tool-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 3%;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  text-decoration: none;
+}
+
+.fa-bars {
+  font-size: 30px;
+  cursor: pointer;
+}
+
+/* Introduction */
+#intro {
+  background-color: #1c1a19;
+  background-image: url("images/backgrounds/bg1_mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 26%;
+  background-position: 100% 115%;
```

Comment:
Kindly you can make the `background-size: 26%;` role a little bit bigger, like `background-size: 35%;`, to make your design like the Figma design.